### PR TITLE
FIX get next/previous conversation not message

### PR DESCRIPTION
**Pull request by misstick**

### CODE REVIEW

File: client/app/stores/message_store.coffee [hunk 1]
```diff
@@ -606,13 +606,18 @@ class MessageStore extends Store
...
 
-        getMessage = ->
-            index0 = messages.toArray().findIndex (message) ->
-                if param.conversationID isnt 'undefined'
-                    return param.conversationID is message.get 'conversationID'
-                return messageID is message.get 'id'
-            index = param.transform index0
-            messages.get index
+        getMessage = =>
+            _newMessage = (index) ->
```
Comment (aenario) [inline client/app/stores/message_store.coffee]: OK, I give up, what is this code suppose to do ?
the `getMessage` function is immediately invoked and not recursive, so seems unnecessary.
the two `index0` will overwrite each other, is that on purpose ?
`messages.toArray().findIndex` -> `message.findEntry`
what does `_newMessage` does exactly ?

Comment (aenario) [inline client/app/stores/message_store.coffee]: On the other hand, if it fixes a bug and we actually do the big refactoring(TM), whatever ;-)

Comment (misstick) [inline client/app/stores/message_store.coffee]: @aenario : Several messages have the same conversationID, so I check if next message isnt port of the same conversation; If you see a better way to do it, I am all ears

Comment (frankrousseau) [inline client/app/stores/message_store.coffee]: That one is hard to understand. Moreover there is a return true without return false in other case. I think comments, explicitly handle all cases and better variable names will improve it.

**Pull request merged**